components/ai_chat/resources/page/components/filter_menu/tools_menu.tsx

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import { ActionEntry, ActionType } from "components/ai_chat/resources/common/mojom";
+import ToolsButtonMenu, { Props } from "./filter_menu";
```

Comment:
Might be clearer if we have a consistent name, both here and in `filter_menu.tsx`, i.e. calling it `FilterMenu`

Reply:
https://github.com/brave/brave-core/pull/29247/commits/fb1670cbeddba88fb004d0062dabbb15ce755201